packages/venia-concept/venia-upward.yml

```diff
@@ -42,15 +42,15 @@ imageOpt:
           engine: mustache
           provide:
             width: $match.$1
-            path: $match.$2
+            path: request.url.query.url
           template:
             when:
               - matches: env.USE_FASTLY
                 pattern: '[^0]'
                 use:
-                  inline: '/{{path}}?auto=webp&format=pjpg&'
+                  inline: '{{path}}?auto=webp&format=pjpg&'
             default:
-              inline: '/{{path}}'
+              inline: '{{path}}'
```

Comment:
What is the affect of these changes? Does something somewhere else need to change to reflect them? I'm assuming this worked at one point 😃

Reply:
Yeah, since we don't have a Fastly-enabled cloud instance to test on (@bbatsche you mentioned you have access to one...) we haven't tested this.